packages/docs/src/lang/en/components/Lazy.json

```diff
@@ -2,6 +2,7 @@
   "heading": "# Lazy loading",
   "headingText": "The `v-lazy` component is used to dynamically load components based upon an elements visiblity.",
   "lazyAlert": "The `v-lazy` component uses the [v-intersect](/directives/intersect) directive which requires a [Polyfill](/directives/intersect#polyfill) in order to work on IE11 / Safari. It's possible on some iOS versions will also require the use of this polyfill.",
+  "usageText": "The `v-lazy` component by default will not render its contents until it has been intersected. Scroll down and watch the element render as you go past it.",
   "examples": {
     "usage": {
       "desc": "The `v-lazy` component by default will not render its contents until it has been intersected. Scroll down and watch the element render as you go past it."
```

Comment:
Forgot to remove?

Reply:
This one didn't end with comma and survived the replacement.
So should leave examples {} or remove examples at all?